src/styles/headerPage.scss

```diff
@@ -7,42 +7,63 @@
   display: flex;
   justify-content: center;
   align-items: center;
-}
 
-.header-items-box {
-  width: 1286px;
-  height: 41px;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: 0 50px;
-}
+  &__items {
+    width: 1286px;
+    height: 41px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
 
-.header-icons-box {
-  height: 24px;
-  width: fit-content;
-  gap: 45px;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+  &__logo {
+    width: 185px;
+    height: 41px;
 
-  & img {
-    max-height: 24px;
+    @include desktop {
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
   }
-}
 
-.header-text-box {
-  width: 430px;
-  height: 24px;
-  display: flex;
-  justify-content: space-between;
-}
+  &__text {
+    width: 430px;
+    height: 24px;
+    display: flex;
+    justify-content: space-between;
+
+    @include desktop {
+      display: none;
+    }
+  }
+
+  &__text--link {
+    text-decoration: none;
+    color: black;
+  }
+
+  &__icons {
+    height: 24px;
+    width: fit-content;
+    gap: 45px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    @include desktop {
+      display: none;
+    }
 
-.header-logo-box {
-  width: 185px;
-  height: 41px;
+    & img {
```

Comment:
Just remember not to apply styles directly to HTML elements if not strictly neccesary